app/javascript/src/utils/api/properties.js

```diff
@@ -6,14 +6,19 @@ export const getProperties = async (params) => {
   const { data } = await axios.get(
     `${ENDPOINT_API}/properties/?q=${params}`
   )
-
   return data
 }
 
 export const getProperty = async (propertyReference) => {
   const { data } = await axios.get(
     `${ENDPOINT_API}/properties/${propertyReference}`
   )
+  return data
+}
 
+export const getAlerts = async (propertyReference) => {
+  const { data } = await axios.get(
+    `${ENDPOINT_API}/cautionary_alerts/properties/${propertyReference}`
+  )
```

Comment:
I would use its own file to integrate it with the API as this is a different endpoint. 😄